### DISCUSSION TITLE
Stop next test if previous test fails

### DISCUSSION THREAD

**Discussion by quad5** [cypress-io/cypress#14209] [CI setup]
So I have created test in the following format. Since the sub sequence test depends on prior test's success, is there a way to stop running the sub sequence test when prior test fails?

> describe(('testing multiple tests'), function() {
    it(('test1'), function() {
        // do something
    })
    it(("test2 which depends on test1's actions "), function() {
        // do something
    })
    it(("test3 which depends on test2's actions "), function() {
        // do something  
    })
})

**Comment by bahmutov** [comment]
Duplicate of https://github.com/cypress-io/cypress/issues/518 see community plugin https://github.com/javierbrea/cypress-fail-fast

**Reply by quad5** [reply]
I should have said that I only want the sub sequence test(s) in `describe` to not execute, not the whole Cypress run. from the sound of the cypress-fail-fast, it sounds like it will stop the whole Cypress run altogether. Or am i wrong?

**Comment by bahmutov** [comment]
You can try it to see

**Reply by quad5** [reply]
I just did and created a feature reuqest [here](https://github.com/javierbrea/cypress-fail-fast/issues/27).
